### PR TITLE
Fixed issue #4

### PR DESCRIPTION
**Pull request by Lekky71**
The share button creates a tweet intent, it has another tweet template text incase winder.username does not exist.
I am yet to implement the automatic loading of results when someone clicks on the link, will do asap, add as an update feature.

### CODE REVIEW

**Comment by Lekky71** [comment]
screen shot 2019-02-16 at 5 31 00 pm

**Comment by shalvah** [comment]
Let's leave the auto loading for a separate PR. One PR per feature.

File: docs/index.html [hunk 1]
```diff
@@ -275,23 +310,41 @@ <h2>#TwitterThrowback</h2>
             loadingIndicator.style.display = 'block';
         }, 1000);
     }
+
+    function shareToTwitter() {
+        let url;
+        if(window.username !== undefined) {
+            url = `http://twitter.com/share?text=Here's my TwitterThrowback. Check yours here` +
```
Comment (shalvah) [inline docs/index.html:321]: Make it a hashtag. "Here's my #TwitterThrowback." You might need to use `&39;` for that .

Oh, never mind. Saw the Twitter option for that.

File: docs/index.html [hunk 2]
```diff
@@ -275,23 +310,41 @@ <h2>#TwitterThrowback</h2>
...
+                `&url=http://oldtweets.today/?username=${window.username}&hashtags=OldTweetsToday,TwitterThrowback`;
+        }
+        else {
+            url = `http://twitter.com/share?text=I just found out this cool site that helps you see your throwback tweets ` +
```
Comment (shalvah) [inline docs/index.html]: Let's make the text "Wanna see what you were tweeting on this day in the past? Check out {URL}."

**Review by shalvah — CHANGES_REQUESTED** [review]
Please merge master into your fork first.

File: docs/index.html (hunk 1 above)
Comment (Lekky71) [reply docs/index.html:321]: I shouldn't bother with &39; ?

File: docs/index.html (hunk 2 above)
Comment (Lekky71) [reply docs/index.html]: Done.

**Comment by Lekky71** [comment]
@shalvah I have made all required changes.

**Comment by shalvah** [comment]
I'd actually have preferred the button to have a text label, but I guess there isn't really good space on the interface.

**Comment by BolajiAyodeji** [comment]
How about a twitter icon?
Should clearly state twitter sharing

File: docs/index.html [hunk 3]
```diff
@@ -137,6 +175,7 @@ <h2>#TwitterThrowback</h2>
     let hash = window.location.hash.split('#')[1];
     if (hash != '' && hash != undefined) {
         usernameInput.value = hash;
+        handleFormSubmit() //todo
```
Comment (shalvah) [inline docs/index.html]: This function is supposed to take an event parameter. Please remove the function call for now. You've done good work here. We can then update the function and add this in a separate PR.

**Review by shalvah — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 3 above)
Comment (Lekky71) [reply docs/index.html]: damn, wanted to remove that, I forgot, lol.

**Comment by Lekky71** [comment]
> I'd actually have preferred the button to have a text label, but I guess there isn't really good space on the interface.

Yes, my thought too.

File: docs/index.html (hunk 3 above)
Comment (Lekky71) [reply docs/index.html]: done

**Comment by shalvah** [comment]
Yes, I think a Twitter icon would be better. The current icon is still confusing to some folks. Plus, iOS uses a different icon, and Android, too, in some places.

**Comment by Lekky71** [comment]
@shalvah done, I also changed the share icon.

**Comment by shalvah** [comment]
Thanks. Good work.

**Comment by Lekky71** [comment]
Thanks

**Pull request merged**